protonmail/auth.go

```diff
@@ -260,6 +260,53 @@ func (c *Client) Unlock(auth *Auth, passphrase string) (openpgp.EntityList, erro
 	c.uid = auth.UID
 	c.accessToken = string(accessTokenBytes)
 	c.keyRing = keyRing
+
+	// Get additional private keys
+	u, err := c.GetCurrentUser()
```

Comment:
I've added `Client.ListAddresses` in a recent commit: https://github.com/emersion/hydroxide/commit/d76f08111417f95610f8e27c7af3aa4c4d20abd9

We should use it instead of `Client.GetCurrentUser`.